teams.md/docs/csharp/getting-started/running-in-teams.md

```diff
@@ -105,14 +99,14 @@ If you want to monitor the activities and events in your app, you can still use
 
 ## Troubleshooting
 
-While Teams AI v2 SDK heavily endorses Teams Tookit (TTK), you can run your agent in Teams without it. However, using TTK saves time and effort.
+In general, we strongly recommend using Microsoft 365 Agents Toolkit for managing your resources, deployment, etc. If you have chosen to manually deploy and publish your resources, please refer to the [Teams app documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/deploy-and-publish/apps-publish-overview) for support. Because Teams AI library does not support resource management, please refer to the [Microsoft Teams deployment documentation](https://learn.microsoft.com/en-us/microsoftteams/deploy-overview).
```

Comment:
For deployment and resource management we recommend the Microsoft 365 Agents Toolkit.
If you prefer to set everything up by hand, follow the standard Teams app publishing guide.
The Teams AI library itself doesn’t handle deployment or Azure resources, so you’ll need to rely on the general Microsoft Teams deployment documentation for that part.